### ISSUE TITLE
Wrong package version for webpack-config-single-spa-ts using create-single-spa command line

### ISSUE THREAD

**Issue by shahabganji** [single-spa/create-single-spa#314]
When creating `root-config` project and `in-browser utility modul` project types using `create-single-spa` command line it creates the project with wrong package version, so it is unable to run the project, the package `webpack-config-single-spa-ts` need to be updated to version `3.0.0`. 

**create the bug**
1. run `create-single-spa` command
2. Follow the steps for creating either `root-config`  or `in-browser utility module`
3. Select Typescript
4. Run the application

You will see the following error: 

```bash

[webpack-cli] Invalid options object. Dev Server has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'firewall'. These properties are valid:
   object { allowedHosts?, bonjour?, client?, compress?, devMiddleware?, headers?, historyApiFallback?, host?, hot?, http2?, https?, ipc?, liveReload?, onAfterSetupMiddleware?, onBeforeSetupMiddleware?, onListening?, open?, port?, proxy?, setupExitSignals?, static?, watchFiles?, webSocketServer? }
 - options.client has an unknown property 'host'. These properties are valid:
   object { logging?, overlay?, progress?, webSocketTransport?, webSocketURL? }
```

**How to resolve**

Update `webpack-config-single-spa-ts` development dependency in `package.json` file to version `3.0.0`.

**Comment by alexristich** [comment]
Hey @shahabganji, this should have been resolved in `create-single-spa@3.0.3`, which @joeldenning released a couple days ago. See https://github.com/single-spa/create-single-spa/issues/309#issuecomment-886191984 for context.

Did this fail with `create-single-spa@3.0.3` or did you create your `root-config` or utility module using an older version?

**Comment by shahabganji** [comment]
@alexristich I had version `3.0.0` installed, after updating to version `3.0.3` the problem is resolved. 👍🏼
